dev/com.ibm.ws.rest.handler.validator/resources/com/ibm/ws/rest/handler/validator/resources/CWWKOMessages.nlsprops

```diff
@@ -34,4 +34,8 @@ CWWKO1551_CANNOT_VALIDATE.useraction=Verify that the feature that provides the c
 
 CWWKO1552_MISSING_DELIMITER=CWWKO1552E: The X-Login-Config-Props header requires an equals sign character (=) as a delimiter between the login configuration property name and value.
 CWWKO1552_MISSING_DELIMITER.explanation=The equals sign delimiter is necessary to determine which part of the X-Login-Config-Props header is the login configuration property name and which part is the value.
-CWWKO1552_MISSING_DELIMITER.useraction=Update the X-Login-Config-Props header to include an equals sign (=) between the login configuration property name and value.
\ No newline at end of file
+CWWKO1552_MISSING_DELIMITER.useraction=Update the X-Login-Config-Props header to include an equals sign (=) between the login configuration property name and value.
+
+CWWKO1553_HANDLER_NOT_FOUND=CWWKO1553E: There are no registered handlers that match the requested URL {0}.
```

Comment:
could we discover the set of available handlers and include them in the error message?

Reply:
Not really, I'm delegating the filtering of /ibm/api/validation and /ibm/api/validation/ to the main rest handler feature that catches invalid endpoints in general.  So our rest handler isn't even being invoked.  For /ibm/api/validation// I'd like to keep the message the same as the other two.